Zilon.Core/Zilon.Bot.Players.DevelopmentTests/BotActorTaskSourceTests.cs

```diff
@@ -25,25 +25,98 @@ public class BotActorTaskSourceTests
+            startUp.RegisterServices(serviceContainer);
+            var serviceProvider = serviceContainer.BuildServiceProvider();
 
             var botSettings = new BotSettings { Mode = mode };
 
             var autoPlayEngine = new AutoplayEngine<HumanBotActorTaskSource>(startUp, botSettings);
-            await autoPlayEngine.StartAsync(_globalServiceProvider).ConfigureAwait(false);
 
-            var scoreManager = _globalServiceProvider.GetRequiredService<IScoreManager>();
+            var startPerson = autoPlayEngine.CreateStartPerson(serviceProvider);
+
+            PrintPersonBacklog(startPerson);
+
+            await autoPlayEngine.StartAsync(startPerson, serviceProvider).ConfigureAwait(false);
+
+            PrintResult(serviceProvider);
+        }
+
+        private void PrintPersonBacklog(HumanPerson humanPerson)
+        {
+            Console.WriteLine("Build In Traits:");
+            var buildinTraits = humanPerson.EvolutionData.Perks.Where(x => x.Scheme.IsBuildIn).ToArray();
+            foreach (var buildInTrait in buildinTraits)
+            {
+                Console.WriteLine(buildInTrait.Scheme.Name.En);
+            }
+
+            Console.WriteLine("Start Equipments:");
+            var equipments = humanPerson.EquipmentCarrier.ToArray();
+            foreach (var equipment in equipments)
+            {
+                if (equipment is null)
+                {
+                    continue;
+                }
+
+                Console.WriteLine(equipment.Scheme.Name.En);
+            }
+
+            Console.WriteLine("Start Inventory:");
+            var inventoryProps = humanPerson.Inventory.CalcActualItems().ToArray();
+            foreach (var prop in inventoryProps)
+            {
+                switch (prop)
+                {
+                    case Equipment equipment:
+                        Console.WriteLine(equipment.Scheme.Name.En);
+                        break;
+
+                    case Resource resource:
+                        Console.WriteLine($"{resource.Scheme.Name.En} x {resource.Count}");
+                        break;
+
+                    default:
+                        Console.WriteLine(prop.Scheme.Name.En);
+                        break;
+                }
+                
+            }
+        }
+
+        private static void PrintResult(ServiceProvider serviceProvider)
+        {
+            var scoreManager = serviceProvider.GetRequiredService<IScoreManager>();
+
             Console.WriteLine($"Scores: {scoreManager.BaseScores}");
 
-            var playerEventLogService = _globalServiceProvider.GetRequiredService<IPlayerEventLogService>();
-            var deathReasonService = _globalServiceProvider.GetRequiredService<DeathReasonService>();
+            var scoreDetails = TextSummaryHelper.CreateTextSummary(scoreManager.Scores);
+            Console.WriteLine($"Details:  {scoreDetails}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5310892)